SPECS/openssl.spec

```diff
@@ -430,10 +437,12 @@ rm -rf %{buildroot}/usr/lib/debug
 %postun libs -p /sbin/ldconfig
 
 %changelog
-* Thu Feb 13 2025 Lin Liu <deli.zhang@cloud.com> - 1.0.2k-26.1
-- CP-53507: Build OpenSSL 1.0.2k-26 compatible package
-
-* Thu Jul 18 2024 Lin Liu <Lin.Liu01@cloud.com> - 1.0.2k-26
-- First imported release
-
+* Thu Aug 07 2025 Andrii Sultanov <andriy.sultanov@vates.tech> - 1.0.2k-26.1.1
+- First build - OpenSSL 1.0.2k-26 compatible package
```

Comment:
Please also mention the added patches here, as this is an important change.

Reply:
Done